docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md

```diff
@@ -1,8 +1,166 @@
+
+Eigen Labs will push through any rewards that are still in the delayed withdrawal router 7 days after the PEPE upgrade (after which point all rewards in there will be claimable). So if you haven’t claimed by this point, we’ll automatically process those claims on your behalf and send them to the wallet of the EigenPod owner.
+
+
+## Prerequisites
+
+The user will need an environment available to run the [EigenPod Proof Gen CLI](https://github.com/Layr-Labs/eigenpod-proofs-generation/tree/master/cli#quickstart) including its software prerequisites.
+
+
+## Key Management and EigenPod Proof Submitter
+
+EigenLayer Native Restaking requires submitting proofs to EigenLayer contracts to prove the amount of validator ETH is active and its withdrawal address is pointing to the EigenPod. For users who do not wish to include the "EigenPod Owner" (aka The EigenPod generation key) in their proof generation commands, you may identify another wallet as the **Proof Submitter** and delegate its privilege to submit proofs on its behalf using the assign_submitter command. This is a **one time process** to assign a submitter for proofs. At any point in the future the `sender` of the proof can be the assigned submitter.
+
+Use the following command to assign a submitter for your EigenPod:
+```bash
+/cli assign-submitter --execNode $NODE_ETH --podAddress $EIGENPOD_ADDRESS --sender $EIGENPOD_OWNER_PK
+```
+
+Consider using a cold key for the EigenPod Owner role. This key should be stored securely and used infrequently. For cold keys, we recommend using hardware wallets (e.g., Ledger, HSMSs) or smart contract multisigs (e.g., Safe). 
+
+We strongly recommend using a seperate key for the Proof Submitter, which can be considered a hot key. The Proof Submitter is any other address approved to submit proofs on behalf of the EigenPod owner. This separation allows the EigenPod owner key to remain secure and cold. Hot keys, while less secure, can be managed with solutions like Vault (Hashicorp) or environment variables. It is crucial not to store any meaningful value in your hot keys as operational keys are considered less secure. 
+
+
+
+## Steps to Deposit (Restake) Validator Native Beacon Chain ETH
+
+### Part 1: Create EigenPod
+
+Call EigenPodManager.createPod() .  
+
+### Part 2: Configure validator(s) withdrawal credentials
+
+1. Configure the validator(s) credentials to point to the EigenPod address when the validator is created. Please see [Ethereum Launchpad](https://launchpad.ethereum.org/en/withdrawals#enabling-withdrawals) for more information. 
+    a. Optional: you may choose to set the FEE_RECIPIENT to your EigenPod address if you wish to Restake those fees.
+
+2. Wait for the validator(s) to become active on-chain. Please see https://beaconcha.in/ to follow your validator status.
```

Comment:
This is good, but we should also tell users to run the CLI `status` command, as it'll tell you whether it "sees" your active validators, which helps you know when you're ready to move on to step 3.

Reply:
Good call. Could you share any example healthy ("seen") output from the `status` command that I could include for users in the docs?

Reply:
I do not have a recent example of a "seen" output because the CLI output has changed a bunch and we currently have a shortage of testing validators :sweat: 

I have an example of the `status` command here _after_ the `credentials` command has been run:

![image](https://github.com/user-attachments/assets/3b6b7deb-08e0-460b-b949-5d12d617ee08)

Reply:
But to answer your question, yes, the `status` command will confirm the withdrawal address is set correctly and the validator is active on the beacon chain. It should be pretty obvious once a user runs it :smile:

Reply:
![image](https://github.com/user-attachments/assets/7e41364d-07ea-428a-ad94-7d4e67c4697d)

I did actually manage to get an up-to-date picture showing there are validators are ready for credential proofs ^

Reply:
Excellent! This will be high value, I'll add it now.